PartitioningAgent/Agent.cs

```diff
@@ -73,6 +82,9 @@ public async Task StartAsync()
 
                     await this.clusterNodes.RemoveStaleNodesAsync();
 
+                    // Scale nodes in vmss
```

Comment:
NIT: captialize.

Reply:
I'd still like to see VMSS capitalized. If you're not going to do this, please say so in this comment.